src/BannerWithLink/index.stories.tsx

```diff
@@ -77,7 +77,9 @@ const StoryTemplate = (args: BrazeMessageProps & { componentName: string }): Rea
                 logButtonClickWithBraze={(internalButtonId) => {
                     console.log(`Button with internal ID ${internalButtonId} clicked`);
                 }}
+                subscribeToNewsletter={() => mockSubscribe('')}
```

Comment:
I think because DCR is passing the newsletter subscribe function through to banner component as well as epic, since we added the newsletter banner? But why it was not picked up before ... I don't know!